pkg/runtime/runtime.go

```diff
@@ -1408,8 +1408,8 @@ func (a *DaprRuntime) publishMessageHTTP(ctx context.Context, msg *pubsubSubscri
 	}
 
 	// Every error from now on is a retriable error.
-	log.Warnf("retriable error returned from app while processing pub/sub event %v: %s. status code returned: %v", cloudEvent[pubsub.IDField], body, statusCode)
-	return errors.Errorf("retriable error returned from app while processing pub/sub event %v: %s. status code returned: %v", cloudEvent[pubsub.IDField], body, statusCode)
+	log.Warnf("retriable error returned from app while processing pub/sub event %v, topic: %v, body: %s. status code returned: %v", cloudEvent[pubsub.TopicField], cloudEvent[pubsub.IDField], body, statusCode)
```

Comment:
The order of params seem to be wrong.

Question: what happens if cloudEvent[pubsub.TopicField] is nil? Will it crash or print `nil`?

Reply:
Print nil, as https://github.com/dapr/dapr/pull/3687/files#R1353.